_elements/buttons.md

```diff
@@ -8,26 +8,55 @@ title: Buttons
   <!-- Add HTML markup for example here -->
 
   <h3>Primary Buttons</h3>
-  <button>Static</button>
-  <button class="usa-button-active">Active</button>
-  <button class="usa-button-hover">Hover</button>
+  <div class="button_wrapper">
+    <button>Sign up</button>
+    <button class="usa-button-active">Sign up</button>
+    <button class="usa-button-hover">Sign up</button>
+  </div>
+
+  <div class="button_wrapper">
+    <button class="usa-button-primary-alt">Sign up</button>
+    <button class="usa-button-primary-alt usa-button-active">Sign up</button>
+    <button class="usa-button-primary-alt usa-button-hover">Sign up</button>
+  </div>
 
   <h3>Secondary Buttons</h3>
-  <button class="usa-button-secondary">Static</button>
-  <button class="usa-button-secondary usa-button-secondary-active">Active</button>
-  <button class="usa-button-secondary usa-button-secondary-hover">Hover</button>
+  <div class="button_wrapper">
+    <button class="usa-button-secondary">Sign up</button>
+    <button class="usa-button-secondary usa-button-active">Sign up</button>
+    <button class="usa-button-secondary usa-button-hover">Sign up</button>
+  </div>
 
-  <h3>Button Focus</h3>
-  <button class="usa-button-focus">Button</button>
+  <div class="button_wrapper">
+    <button class="usa-button-gray">Sign up</button>
+    <button class="usa-button-gray usa-button-active">Sign up</button>
+    <button class="usa-button-gray usa-button-hover">Sign up</button>
+  </div>
 
-  <h3>Outlined Button</h3>
-  <button class="usa-button-outlined" type="button">Button</button>
+  <div class="button_wrapper">
+    <button class="usa-button-outlined" type="button">Sign up</button>
+    <button class="usa-button-outlined usa-button-active">Sign up</button>
+    <button class="usa-button-outlined usa-button-hover">Sign up</button>
+  </div>
+
+  <div class="button_wrapper" style="background: #323a45">
```

Comment:
The dark wrapper extends too long. Can you make sure it extends up to the third button, not the full width of the container?

Here's the design:

![dark wrapper](https://cloud.githubusercontent.com/assets/5249443/9308619/f71eca94-44b9-11e5-8ed8-a28bf36999c4.png)